library/src/main/java/com/mux/player/media/MediaItems.kt

```diff
@@ -143,6 +147,8 @@ object MediaItems {
     maxResolution: PlaybackResolution? = null,
     minResolution: PlaybackResolution? = null,
     renditionOrder: RenditionOrder? = null,
+    assetStartTime: Double? = null,
```

Comment:
In the future will it be possible to prevent callers from setting both `assetStartTime`/`assetEndTime` and `programStartTime`/`programEndTime`? We currently only support one or the other being set.

Reply:
Sure, I think that'd be fine. In the future when Player offers documented support for that feature, we can discuss either handling that, or verbosifying the `MediaItems` API. 

I don't consider this a concern currently